icubam/backoffice/handlers/plot.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+import tornado.web
+import math
+
+from bokeh.embed import components
+from bokeh.plotting import figure
+from icubam.backoffice.handlers import base
+
+from pprint import pprint
+
+
+class PlotHandler(base.BaseHandler):
+  ROUTE = '/plot'
+
+  def initialize(self, config, db):
+    super().initialize(config, db)
+
+  @tornado.web.authenticated
+  async def get(self):
+    """Serves a page with a table gathering current bedcount data with some extra information."""
+    data = {}
+    figures = []
+
+    # plot
+    # bed_counts = self.db.get_bed_counts()
+    bed_counts = self.db.get_visible_bed_counts_for_user(None, force=True)
+    cities = {}
```

Comment:
what about a collections.defaultdict(int) ? that would simplify the loop below with a single line
cities[curr.icu.city] += curr.n_covid_free

Reply:
Good point!